src/NexyCrypt.php

```diff
@@ -320,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = ['verify' => false])
+    private function request($method, $uri, array $options = ['verify' =>  __DIR__.'/cacert.pem'])
```

Comment:
Please see: https://github.com/nexylan/nexycrypt/pull/4#discussion_r81512174

And I'm not talking about the filename but the option name.

`verify` is not very appropriate for a path.